keanu-project/src/test/java/io/improbable/keanu/network/BayesianNetworkTest.java

```diff
@@ -90,4 +97,17 @@ public void labelErrorsDetected() {
 
         BayesianNetwork net = new BayesianNetwork(a.getConnectedGraph());
     }
+
+    @Test
+    public void youCanSaveAndLoadANetworkWithValues() throws IOException {
+        DoubleVertex gaussianVertex = new GaussianVertex(0.0, 1.0);
+        BayesianNetwork net = new BayesianNetwork(gaussianVertex.getConnectedGraph());
+        ByteArrayOutputStream output = new ByteArrayOutputStream();
+
+        ProtobufWriter protobufWriter = new ProtobufWriter(net);
+        protobufWriter.save(output, true);
+        assertThat(output.size(), greaterThan(0));
+        ByteArrayInputStream input = new ByteArrayInputStream(output.toByteArray());
+        net = ProtobufReader.loadNetwork(input);
```

Comment:
can you assert something about the `net`?

Reply:
Done now.